website/static/js/registrationUtils.js

```diff
@@ -779,7 +781,54 @@ RegistrationEditor.prototype.save = function() {
     }
     return true;
 };
+/**
+ * Returns the `user_fullname` of each contributor attached to a node.
+ **/
+RegistrationEditor.prototype.getContributors = function() {
```

Comment:
To improve maintainability of code, we're trying to keep access to data separate from manipulation of that data. For now, can you please create a separate function to do the network request (and return a promise)? 

This function can then define the `.done` handler that does the data transformation.